packages/rest/src/lib/utils/constants.ts

```diff
@@ -27,6 +27,9 @@ export const DefaultRestOptions = {
 	hashSweepInterval: 14_400_000, // 4 Hours
 	hashLifetime: 86_400_000, // 24 Hours
 	handlerSweepInterval: 3_600_000, // 1 Hour
+	async makeRequest(...args) {
+		return fetch(...args);
```

Comment:
Don't we want to default to undici request given its the best performing? It's a matter of deciding which one is worse: "by default, the package does not work on web without specifying this option" or "by default, the performance of this package is significantly worse for the majority of our userbase (nodejs) without specifying this option".

Personally, I feel like the former should be the way to go.

Reply:
I would love to do that, but we simply can't. If we export with a default of undici, then it imports it unnecessarily in edge / web which though it is possible it can get tree-shook, would likely end up throwing.
Also, in all likelihood, we will make undici the default in /next, because that is totally possible while leaving this the way it is

Reply:
You can certainly do it by making undici a dynamic `import()` wrapped with `lazy`, and accessing it that way.